test/test_torch.py

```diff
@@ -197,7 +197,6 @@ def test_namespace(ns, *skips):
                        'index_put',
                        'is_coalesced',
                        'is_distributed',
```

Comment:
is this a list of all docs that we don't have?

Reply:
I suppose so. I haven't checked this.